src/cascadia/TerminalApp/LeafPane.h

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+#include <winrt/Microsoft.Terminal.TerminalControl.h>
+#include <winrt/TerminalApp.h>
```

Comment:
Do we really need these? Aren't they already in the pch? Shouldn't they be in the pch?

Reply:
Yep, don't need them. Removed!